src/todo/UI.lua

```diff
@@ -222,16 +246,23 @@ function todo.create_add_edit_frame(player, task)
             name = "todo_persist_button",
             caption = {"todo.persist"}
         })
-        flow.add({
-            type = "checkbox",
-            style = "todo_checkbox_default",
-            name = "todo_add_top",
-            state = false,
-            caption = {"todo.add_top"}
-        })
     end
 end
 
+function todo.create_delete_confirmation_button(element, id)
+    local table = element.parent
+    todo.log(table)
+    element.destroy()
+
+    table.add({
+        type = "button",
+        style = "todo_button_default",
+        name = "todo_confirm_deletion_button_" .. id,
+        caption = {"todo.confirm_deletion"}
+    })
```

Comment:
We should have a cancel button too, shouldn't we?

Reply:
Then maybe I'm wrong, the table may already have a cancel button...

Reply:
Yeah.. I wanted to use the one from the edit dialog. The deletion is simply just a two step process to prevent accidental deletions. You have to click twice (even though it is the same sport).